### PR TITLE
Build a RWD portofolio version

### PR DESCRIPTION
**Pull request by amentetteh**
Open for :

- Update style.css and add media queries and styles to build mobile-first version
- Add backgroung images
- Update index.html to fit desktop version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -229,23 +229,29 @@ <h2 class="primary-header">About me</h2>
     <section class="contact-form">
       <h2 id="contact-header-message"> I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
       <form action="https://formspree.io/f/xgeqyrjy" method="post">
+        <div id="first-last-name">
+          <input class="field" name="firstName" id="firstName" type="text" placeholder="First Name" maxlength="30" autocomplete="off" required>
+          <input class="field" name="lastName" id="lastName" type="text" placeholder="Last Name" maxlength="30" autocomplete="off" required>
+        </div>
```
Comment (Santiago220991) [inline index.html]: What a great job. One of the best projects I have seen. :clap: :clap:  However, there is just one tiny issue. Kindly make sure that your contact form in the desktop version is still working. :pray: :pray:  It's not necessary to use two inputs for the name and last name It's ok if you just use one as the mobile version. This is mentioned in the requirements of the project. When you apply this change the contact form stops working.  
image
image

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @amentetteh, 
I'm Santiago your reviewer for this PR. :wave: :wave: 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
:heavy_check_mark: Excellent design.
:heavy_check_mark: Good PR summary.
:heavy_check_mark: Meaninfull branch feature name.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (amentetteh) [reply index.html]: Ok. Great! Thanks!

**Review by sumairq — APPROVED** [review]
### Hi @amentetteh  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**